src/App.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
+ import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Home from './pages/Home';
+import About from './pages/About';
+import Quiz from './pages/Quiz';
+import Profile from './pages/Profile';
+import Error404 from './pages/Error/Error404';
 
 function App() {
   return (
-    <div>
-      <h1 className="text-2xl bg-lightblue">The Movie Bubble</h1>
-    </div>
+    <>
+      <BrowserRouter>
+        <Switch>
+          <Route exact path="/">
```

Comment:
although this method works, react router `Route` component doesn't accept children. Which means there shouldn't be anything between `<Route>` and `</Route>` not even a space

fix it like this

```jsx
<Route path="/about" component={About} />
```

or if you need to pass props

```jsx
<Route path="/about" render={() => <About someProp={someVal} /> } />
```